### ISSUE TITLE
Button 'Zoom out' is missing

### ISSUE THREAD

**Issue by knilde** [osmandapp/OsmAnd#12570]
My custom profile for hiking lost the on-screen-button for ZOOM OUT.
It happened after export > fresh install > reimport.

I could not find anny in-app setting for show/hide that button.

How to get it back?
Guess there is some setting in sqlite or the exported file I need to know

OsmAnd Version: OsmAnd+ 3.9.10
Android/iOS version: 7
Device model: Samsung S7

**Comment by pebogufi** [comment]
You mean minus button (-) ?
Is at his position something different ?
What about (+), is it visible ?
Please upload screenshot

**Comment by vshcherb** [comment]
Sounds like broken installation, let's wait for a screenshot.

**Comment by knilde** [comment]
See the attached screenshot.
It only happens with this profile. All other profiles are OK.
![Screenshot_20210824-001516](https://user-images.githubusercontent.com/79543337/130654734-995bb476-68b2-4ad5-ba7a-e75867d3dcb5.png)

Also I attached the profile itself.
I investigated the content but could not find any parameter for that button.
[wandern.osf.zip](https://github.com/osmandapp/OsmAnd/files/7040679/wandern.osf.zip)
